code/modules/atmospherics/machinery/datum_pipeline.dm

```diff
@@ -85,11 +85,11 @@
 
 /datum/pipeline/proc/addMachineryMember(obj/machinery/atmospherics/components/considered_component)
 	other_atmosmch |= considered_component
-	var/datum/gas_mixture/component_mixture = considered_component.returnPipenetAir(src)
-	if(!component_mixture)
+	var/list/returned_airs = considered_component.returnPipenetAir(src)
+	if (!returned_airs.len)
```

Comment:
Using length() would be better here, it also occurs to me, the only case where this would fire is if the component isn't connected to the pipeline. Maybe do something like `if(!length(returned_airs) || (null in returned_airs))` to check? it's slightly slower, but it's worth the sanity check I think